src/Frame/VisualizerFrame.py

```diff
@@ -16,6 +16,8 @@ def __init__(self, root, pathToSummaryCsv, chosenCols):
         self.root = root
         self.df_windowed = DataFrame_Windowed(pathToSummaryCsv, chosenCols)
         self.SetupGraphs()
+        self.patientId = self.fileSelectorFrame.GetPatientId()
```

Comment:
You'll need to change the init function for this class too.
```suggestion
        self.patientId = patientId
```